### PR TITLE
Add new category for API gateway/management tools

### PR DESCRIPTION
**Pull request by pursultani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,6 +215,17 @@ the Design of Network-based Software Architectures](https://www.ics.uci.edu/~fie
 * [raml2html](https://github.com/raml2html/raml2html) - Generates HTML documentation from a RAML file.
 * [ReDoc](https://github.com/Rebilly/ReDoc/) - OpenAPI/Swagger-powered three-panel documentation.
 
+## Management
```
Comment (fzaninotto) [inline README.md]: Rename to API Gateway

File: README.md [hunk 2]
```diff
@@ -214,6 +215,17 @@ the Design of Network-based Software Architectures](https://www.ics.uci.edu/~fie
...
+
+
+* [Kong](https://github.com/Kong/kong) - Scalable, distributed, and plugin oriented API gateway backed by Nginx.
+* [Tyk API Gateway](https://github.com/TykTechnologies/tyk) - Lightweight API gateway with analytics logging, written in Go.
+* [KrakenD](https://github.com/devopsfaith/krakend) - High performance API gateway with middlewares, written in Go.
+* [apiman](https://github.com/apiman/apiman) - Standalone and embeddable API management with web interface, written in Java.
+* [API Umbrella](https://github.com/NREL/api-umbrella) - API management platform for exposing web services, with web interface and analytics, written in Lua.
+* [WSO2 API Management](https://github.com/wso2/product-apim) - API management tool with lightweight gateway and API lifecycle manangement, written in Java.
+* [3scale APIcast](https://www.3scale.net/) - Nginx based API gateway to integrate internal and external API services with 3scale's API Management Platform.
```
Comment (fzaninotto) [inline README.md]: To be moved to SaaS tools below

File: README.md [hunk 3]
```diff
@@ -214,6 +215,17 @@ the Design of Network-based Software Architectures](https://www.ics.uci.edu/~fie
 * [raml2html](https://github.com/raml2html/raml2html) - Generates HTML documentation from a RAML file.
 * [ReDoc](https://github.com/Rebilly/ReDoc/) - OpenAPI/Swagger-powered three-panel documentation.
 
+## Management
+
+
+* [Kong](https://github.com/Kong/kong) - Scalable, distributed, and plugin oriented API gateway backed by Nginx.
+* [Tyk API Gateway](https://github.com/TykTechnologies/tyk) - Lightweight API gateway with analytics logging, written in Go.
+* [KrakenD](https://github.com/devopsfaith/krakend) - High performance API gateway with middlewares, written in Go.
+* [apiman](https://github.com/apiman/apiman) - Standalone and embeddable API management with web interface, written in Java.
```
Comment (fzaninotto) [inline README.md]: seems to be dead, please remove

File: README.md [hunk 4]
```diff
@@ -214,6 +215,17 @@ the Design of Network-based Software Architectures](https://www.ics.uci.edu/~fie
 * [raml2html](https://github.com/raml2html/raml2html) - Generates HTML documentation from a RAML file.
 * [ReDoc](https://github.com/Rebilly/ReDoc/) - OpenAPI/Swagger-powered three-panel documentation.
 
+## Management
+
+
+* [Kong](https://github.com/Kong/kong) - Scalable, distributed, and plugin oriented API gateway backed by Nginx.
+* [Tyk API Gateway](https://github.com/TykTechnologies/tyk) - Lightweight API gateway with analytics logging, written in Go.
+* [KrakenD](https://github.com/devopsfaith/krakend) - High performance API gateway with middlewares, written in Go.
```
Comment (fzaninotto) [inline README.md]: A bit less popular than the others, please remove.

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (hpsultani) [reply README.md]: Although this one is not very popular, it's the only one that can be embedded. Besides, its community still contributing to it. Do you want to remove it?

**Comment by pursultani** [comment]
Sorry I couldn't fix these earlier, had a busy week.

I left a note on apiman. Please let me know what you think.

**Review by fzaninotto — APPROVED** [review]

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**